ManatiCR/retopais
# RP-3: Problems page

https://manati.atlassian.net/browse/RP-3
## Steps to test
- `drush fra -y && drush cc all`
- Visit `problematicas`. Check that all styles are being applied correctly.
